src/software/backend/input/network/networking/proto_multicast_listener.tpp

```diff
@@ -62,12 +64,20 @@ void SSLVisionClient::handleDataReception(const boost::system::error_code& error
         // Start listening again to receive the next data
         socket_.async_receive_from(
             boost::asio::buffer(raw_received_data_, max_buffer_length), sender_endpoint_,
-            boost::bind(&SSLVisionClient::handleDataReception, this,
+            boost::bind(&ProtoMulticastListener::handleDataReception, this,
                         boost::asio::placeholders::error,
                         boost::asio::placeholders::bytes_transferred));
 
         LOG(WARNING)
-            << "An unknown network error occurred when attempting to receive SSL Vision Data. The boost system error code is "
+            << "An unknown network error occurred when attempting to receive ReceiveProto Data. The boost system error code is "
             << error << std::endl;
     }
+
+    if (num_bytes_received == max_buffer_length)
```

Comment:
why is this a hard check? what if we get 9001 bytes?

Reply:
it's not possible, but I'll change it then

Reply:
Sorry, could you clarify why this isn't possible?

Reply:
well how can you receive more bytes than you have room in the buffer for? I've done some testing and == worked fine

Reply:
Ah, I understand. Since `==` and `>=` are equivalent here I'd tend towards `>=` for clarity, but you're absolutely correct.